aws-amplify/amplify-ui
# Add canonicalUrlPath to help set a custom canonical url

#### Description of changes
- Add a new frontmatter property, `canonicalUrlPath` allow setting the canonical url for a specific page. By doing this, pages with duplicated content across different supported frameworks point to a specific page to denote it as the canonical url
- Use `canonicalUrlPath` for pages that have duplicated content across the list of supported frameworks



#### Issue #, if available



#### Description of how you validated changes

Staging site: https://canonical-urls.d1qle6zdmkrzti.amplifyapp.com/

To verify:
1. Go to https://canonical-urls.d1qle6zdmkrzti.amplifyapp.com/angular/getting-started/introduction
2. Inspect the page in developer tools and find the canonical link inside the `head` tag and verify that it sets the href to be `.../react/getting-started/introduction
3. Try this with other supported frameworks too

#### Checklist



- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
